sdk/servicebus/Azure.Messaging.ServiceBus/tests/Samples/Sample08_Administration.cs

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Identity;
+using NUnit.Framework;
+using Azure.Messaging.ServiceBus.Administration;
+
+namespace Azure.Messaging.ServiceBus.Tests.Samples
+{
+    public class Sample08_Administration : ServiceBusLiveTestBase
+    {
+        /// <summary>
+        /// Authenticate with <see cref="DefaultAzureCredential"/>.
+        /// </summary>
+        public void AuthenticateWithConnectionString()
+        {
+            #region Snippet:ServiceBusAdministrationClient
+            // Create a ServiceBusAdministrationClient that will authenticate using a connection string
+            string connectionString = "<connection_string>";
+            ServiceBusAdministrationClient client = new ServiceBusAdministrationClient(connectionString);
+            #endregion
+        }
+
+        /// <summary>
+        /// Create subscription with administration client <see cref="ServiceBusAdministrationClient"/>.
+        /// </summary>
+        public async Task CreateSubscriptionWithAdminisatrationClient()
```

Comment:
I don't think this is necessary since we already have this [sample](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample07_CrudOperations.md#create-a-topic-and-subscription).